src/reanimated2/ViewDescriptorsSet.ts

```diff
@@ -11,82 +10,37 @@ export interface ViewRefSet<T> {
 }
 
 export interface ViewDescriptorsSet {
-  batchToRemove: Set<number>;
-  tags: Set<number>;
-  waitForInsertSync: boolean;
-  waitForRemoveSync: boolean;
   sharableViewDescriptors: SharedValue<Descriptor[]>;
-  items: Descriptor[];
   add: (item: Descriptor) => void;
   remove: (viewTag: number) => void;
-  rebuildsharableViewDescriptors: (
-    sharableViewDescriptor: SharedValue<Descriptor[]>
-  ) => void;
 }
 
-const scheduleUpdates = shouldBeUseWeb() ? requestAnimationFrame : setImmediate;
-
 export function makeViewDescriptorsSet(): ViewDescriptorsSet {
-  const ref = useRef<ViewDescriptorsSet | null>(null);
-  if (ref.current === null) {
-    const data: ViewDescriptorsSet = {
-      batchToRemove: new Set(),
-      tags: new Set(),
-      waitForInsertSync: false,
-      waitForRemoveSync: false,
-      sharableViewDescriptors: makeMutable([]),
-      items: [],
-
-      add: (item: Descriptor) => {
-        if (data.tags.has(item.tag)) {
-          return;
-        }
-        data.tags.add(item.tag);
-        data.items.push(item);
-
-        if (!data.waitForInsertSync) {
-          data.waitForInsertSync = true;
-
-          scheduleUpdates(() => {
-            data.sharableViewDescriptors.value = data.items;
-            data.waitForInsertSync = false;
-          });
+  const sharableViewDescriptors = makeMutable<Descriptor[]>([]);
```

Comment:
Can we rename `sharable` to `shareable` here and everywhere else?

Reply:
I'd prefer not to as a part of this PR to avoid making it even more bloated. This name is also referenced in other places that I didn't touch (12 occurrences thorough the codebase)